src/ServiceControl.AcceptanceTests/ConfigureEndpointAzureServiceBusNetStandardTransport.g.cs

```diff
@@ -29,7 +29,7 @@ public Task Cleanup()
         return Task.FromResult(0);
     }
 
-    public string Name => "AzureServiceBus .NET Standard";
+    public string Name => ServiceControlInstaller.Engine.Instances.TransportNames.AzureServiceBus;
```

Comment:
How about adding

```
using ServiceControlInstaller.Engine.Instances
```

Reply:
Done. Resharper wouldn't do it automatically (maybe because of the extern statement?) and I forgot to fix it before I pushed.